### PR TITLE
feat: add rtl menu

### PR DESCRIPTION
**Pull request by xrkffgg**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.
- https://github.com/vueComponent/ant-design-vue/issues/2038
### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)
- Menu
### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by xrkffgg** [comment]
@tangjinzhou 

I need you update `vc-menu` form `rc-menu`. It is best to be up to date.

**Comment by xrkffgg** [comment]
when you done, i will rebase this branch

File: components/menu/MenuItem.jsx [hunk 1]
```diff
@@ -25,6 +27,7 @@ export default {
       title: title || (level === 1 ? $slots.default : ''),
     };
     const siderCollapsed = this.layoutSiderContext.sCollapsed;
+    const direction = this.configProvider.direction;
```
Comment (Amour1688) [inline components/menu/MenuItem.jsx:30]: 这里用解构比较好吧

File: components/menu/index.jsx [hunk 2]
```diff
@@ -224,6 +224,7 @@ const Menu = {
     const { getPopupContainer: getContextPopupContainer } = this.configProvider;
     const { prefixCls: customizePrefixCls, theme, getPopupContainer } = this.$props;
     const getPrefixCls = this.configProvider.getPrefixCls;
```
Comment (xrkffgg) [inline components/menu/index.jsx:226]: @Amour1688 之前有一个 getPrefixCls 这么写了，然后我就统一这样写了😂😂

Comment (xrkffgg) [reply components/menu/index.jsx:226]: 如果要用解构，还要改之前的，所以就没动，保持统一。

看以后是不是需要一起改

Comment (tangjinzhou) [reply components/menu/index.jsx:226]: 后面改的都用结构吧

Comment (xrkffgg) [reply components/menu/index.jsx:226]: 嗯，新写的我都用了解构了，之前的我都改一下

Comment (xrkffgg) [reply components/menu/index.jsx]: @tangjinzhou has fixed

**Comment by xrkffgg** [comment]
Close first, reopen when necessary.

**Pull request closed without merging**